### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Aditya94A** [bluelinelabs/LoganSquare#210]

**Comment by miomao656** [comment]
it seems so...

**Comment by kihaki** [comment]
Well, you could create a clone and keep on developing, if you do, please post it here!

**Comment by mannodermaus** [comment]
Eric has stated that he's unfortunately unable to dedicate as much time to the project at this time, with it being a side project and all. However, we're currently on the verge of figuring out an approach for its succession, though!

**Comment by EricKuck** [comment]
Yep, what @aurae said. Development has definitely slowed down. Partially due to time and partially due to project maturity. You should see more soon, as we're trying to make the library more of a collaborative project instead of a one-man show.

**Comment by re-thc** [comment]
Is  anyone going to take this up to maintain it? Are there any plans? Happy to help.
